tools/hcluster_sg_parser/hcluster_sg_parser.py

```diff
@@ -8,15 +8,19 @@
 python hcluster_sg_parser.py [-m <N>] [-M <N>] <file> <discarded_out>
 """
 import optparse
+import os
 import sys
 
 
 def main():
     parser = optparse.OptionParser()
     parser.add_option('-m', '--min', type='int', default=0, help='Minimum number of cluster elements')
     parser.add_option('-M', '--max', type='int', default=sys.maxsize, help='Maximum number of cluster elements')
+    parser.add_option('-d', '--dir', type='string', default="", help="Absolute or relative path to output directory. If directory does not exist it will be created")
```

Comment:
```suggestion
    parser.add_option('-d', '--dir', type='string', help="Absolute or relative path to output directory. If the directory does not exist, it will be created")
```